functional-test/src/main/java/org/zanata/page/WebDriverFactory.java

```diff
@@ -107,11 +108,17 @@ private WebDriver createPlainDriver() {
     }
 
     private WebDriver configureChromeDriver() {
+        // TODO can we use this? it will use less code, but it will use DISPLAY rather than webdriver.display
+//        System.setProperty(ChromeDriverService.CHROME_DRIVER_LOG_PROPERTY,
```

Comment:
On 2014-12-23 09:30, Patrick Huang wrote:
> In functional-test/src/main/java/org/zanata/page/WebDriverFactory.java
> <https://github.com/zanata/zanata-server/pull/655#discussion-diff-22197029>:
> 
>> @@ -107,11 +108,17 @@ private WebDriver createPlainDriver() {
>>      }
>>  
>>      private WebDriver configureChromeDriver() {
>> +        // TODO can we use this? it will use less code, but it will use DISPLAY rather than webdriver.display
>> +//        System.setProperty(ChromeDriverService.CHROME_DRIVER_LOG_PROPERTY,
> 
> DISPLAY is global env variable right? This webdriver.display property is
> to enable webdriver to run in its own display such as xnest or Xephyr.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/655/files#r22197029>.
> 

$DISPLAY is an env variable yes, but there are ways of changing it.  And
it is the standard variable which is automatically set by tools like the
Xvfb/Xvnc plugins for Jenkins.

Is there any reason we can't just use $DISPLAY for choosing Selenium's X
display in our builds?


-- 
Sean Flanigan

Senior Software Engineer
Engineering - Internationalisation
Red Hat